package.json

```diff
@@ -17,33 +17,32 @@
         "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js"
     },
     "dependencies": {
-        "@11ty/eleventy": "0.12.1",
-        "@11ty/eleventy-navigation": "0.2.0",
-        "fluid-binder": "1.1.2",
+        "@11ty/eleventy": "1.0.2",
+        "@11ty/eleventy-navigation": "0.3.5",
+        "fluid-binder": "2.0.0",
         "fluid-location-bar-relay": "1.0.7",
         "foundation-sites": "5.5.2",
-        "highlight.js": "11.4.0",
-        "infusion": "4.0.0-dev.20220209T141225Z.50bdbf8d1.main",
+        "highlight.js": "11.6.0",
+        "infusion": "4.5.0",
         "lunr": "2.3.9",
-        "markdown-it-anchor": "8.4.1",
-        "octicons": "3.1.0",
```

Comment:
I'm not sure what the cause is, but I got an error running npm run build or npm run start complaining that markdown-it itself was not installed. 
````
[dev  ] [11ty] Eleventy CLI Fatal Error: (more in DEBUG output)
[dev  ] [11ty] 1. Error in your Eleventy config file 'E:/source/gits/infusion-docs/.eleventy.js'. You may need to run `npm install`. (via EleventyConfigError)
[dev  ] [11ty] 2. Cannot find module 'markdown-it'
````

Adding the line here:

        "markdown-it": "13.0.1",

made the problem go away